pkg/azureutils/azure_disk_utils.go

```diff
@@ -1416,7 +1416,24 @@ func isFatalNetError(err error) bool {
 }
 
 func isNetError(err error) bool {
-	return net.IsConnectionRefused(err) || net.IsConnectionReset(err) || net.IsTimeout(err) || net.IsProbableEOF(err)
+	return net.IsConnectionRefused(err) || net.IsConnectionReset(err) || net.IsTimeout(err) || net.IsProbableEOF(err) || isOtherNetError(err)
+}
+
+func isOtherNetError(err error) bool {
+	if err == nil {
+		return false
+	}
+
+	msg := err.Error()
+
+	switch {
+	case strings.Contains(msg, "broken pipe"):
```

Comment:
case insensitive string comparison might be better here, Using `strings.ToLower(msg)`?

Reply:
These are hardcoded error strings which are not going to change, so a case-insensitive comparison is unnecessary here. This is consistent with how K8s checks for specific error message, see [IsProbableEOF](https://github.com/kubernetes/apimachinery/blob/v0.27.4/pkg/util/net/http.go#L79).